### PR TITLE
Initial content

### PR DESCRIPTION
**Pull request by frenzymadness**

### CODE REVIEW

**Comment by frenzymadness** [comment]
It seems that the tool itself is compatible with Python 2. The tests on the other side will require some work.

**Comment by frenzymadness** [comment]
Now, both files have complete history and tests are passing (except Python 3.12, which is not available yet).

File: README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+This tool has been in CPython since version [2.0 (August 1994)](https://github.com/python/cpython/commit/9af22a037fc961b51651324bdd17a43567a688fd)
+until [October 2022 and version 3.12](https://github.com/python/cpython/commit/e0ae9ddffe0a708d0d3f5b8cc10488d466fc43c4).
+
+Because we (Python maintenance team in Red Hat) use this tool in Fedora Linux to [fix shebangs in RPM packages](https://src.fedoraproject.org/rpms/python-rpm-macros/blob/rawhide/f/macros.python#_21)
+we decided to create a new upstream for it and allow other users to use it and contribute to it.
+
+# License
+
+PSF
```
Comment (hroncok) [inline README.md]: Suggestion: Let's use SPDX here. We are migrating all Fedora packages to it anyway.

```suggestion
[Python-2.0.1](https://spdx.org/licenses/Python-2.0.1.html)
```

**Review by hroncok — APPROVED** [review]
Looking great.

**Comment by frenzymadness** [comment]
Suggestion commited and squashed. I'm going to merge this and add a tag so we can reference it in the macros. CI will follow.

**Pull request merged**